docs/configure_open_edx.md

```diff
@@ -83,3 +83,18 @@ mitxpro-oauth2`
     - `EXTERNAL_XPRO_HOST` will depend on your OS, but it needs to be resolvable within the edx container
         - Linux users: The gateway IP of the docker-compose networking setup for xPro as found via `docker network inspect mitxpro_default`
         - OSX users: Use `host.docker.internal`
+
+
+
+#### Configure Open edX to support OAuth2 authentication from xPro
+
+  - In Open edX:
+    - go to `/admin/oauth2_provider/application/` and create a new application with these settings selected:
+      - `Redirect uris`: `http://<EXTERNAL_XPRO_HOST>:8053/login/_private/complete`
+      - `Client type`: "Confidential"
+      - `Authorization grant type`: "Authorization code"
+      - `Skip authorization`: checked
+      - Other values are arbitrary but be sure to fill them all out. Save the client id and secret for later
+  - In xPro:
+    - Set `OPENEDX_API_CLIENT_ID` to the client id
+    - Set `OPENEDX_API_CLIENT_SECRET` to the client secret
```

Comment:
As we discussed earlier, OSX users will need to include the redirect URIs for both the aliased xpro host and the external xpro host for the OAuth2 Application in XPro, i.e.:

```
http://edx.odl.local:18000/auth/complete/mitxpro-oauth2/
http://host.docker.internal:18000/auth/complete/mitxpro-oauth2/
```

Reply:
This still needs to be addressed. The change would be made here: https://github.com/mitodl/mitxpro/pull/94/files#diff-5a800b453d8b3cf49fd30b01eda6fc1dR32

Reply:
More specifically, this is the change to the instructions that would reflect what I had to do to get this working in OSX:

```
#### Configure xPro as a OAuth provider for Open edX

In xPro:

- go to `/admin/oauth2_provider/application/` and create a new application with these settings selected:
  - `Redirect uris`: `http://<EDX_HOSTNAME>:18000/auth/complete/mitxpro-oauth2/`
    - _[OSX users]_ You will need redirect uris for both the local edX host alias and for `host.docker.internal`. This value should be: `http://edx.odl.local:18000/auth/complete/mitxpro-oauth2/ http://host.docker.internal:18000/auth/complete/mitxpro-oauth2/`
```